src/vs/workbench/contrib/quickaccess/browser/commandsQuickAccess.ts

```diff
@@ -126,11 +125,6 @@ export class CommandsQuickAccessProvider extends AbstractEditorCommandsQuickAcce
 	}
 
 	protected getCommandPicks(token: CancellationToken): Array<ICommandQuickPick> {
```

Comment:
I wonder if we need to introduce another picker behaviour where picks show instantly and slower picks show in a new category. The reason I bring this up: today with additional picks enabled, you need to wait `200ms` for the fast picks. For commands I would revisit that and ensure fast picks are instant because we know we put the slow picks in a separate category anyway.